frontend/packages/console-app/src/components/cloud-shell/CloudShellTab.scss

```diff
@@ -0,0 +1,16 @@
+.co-cloud-shell-tab {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  background-color: #000;
```

Comment:
can we use pf variables for `background-color` as well

Reply:
updated